atlasdb-jdbc/src/main/java/com/palantir/atlasdb/keyvalue/jdbc/JdbcTimestampBoundStore.java

```diff
@@ -38,7 +38,7 @@
     private final Table<Record> TABLE;
     private static final Field<Integer> DUMMY_COLUMN = DSL.field("dummy_column", Integer.class);
     private static final Field<Long> LATEST_TIMESTAMP = DSL.field("latest_timestamp", Long.class);
-    private static final TableReference TIMESTAMP_TABLE = TableReference.createWithEmptyNamespace("_timestamp");
+    static final TableReference TIMESTAMP_TABLE = TableReference.createWithEmptyNamespace("_timestamp");
```

Comment:
Somehow it feels like this should be using `AtlasDbConstants.TIMESTAMP_TABLE`, or rather, all instances of this should be replaced with the default one

Reply:
True, but I think we will drop support for Jdbc at some point so feel that this refactor might be unecessary.

Reply:
Sure, it's true that we do plan on killing this off at some point 😃